test/e2e/network/network_policy.go

```diff
@@ -906,6 +951,94 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 			testCannotConnect(f, f.Namespace, "client-a", service, allowedPort)
 		})
 
+		ginkgo.It("should work with Ingress,Egress specified together [Feature:NetworkPolicy]", func() {
+			const allowedPort = 80
+			const notAllowedPort = 81
+			protocolUDP := v1.ProtocolUDP
+
+			nsBName := f.BaseName + "-b"
+			nsB, err := f.CreateNamespace(nsBName, map[string]string{
```

Comment:
Need to clean up namespace.

Reply:
There is no need to cleanup namespace, it will be cleaned up automatically. https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/framework.go#L485 

Example from log:
```
STEP: Destroying namespace "network-policy-2306" for this suite.
STEP: Destroying namespace "network-policy-b-6489" for this suite.
```